gofundme/src/component/Forms.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+class Form extends React.Component{
+    state = {
+        name: '',
+        captions: '',
+        value: 0
+    }
+    handleChange =(e)=>{
+        this.setState({
+            [e.target.name]: e.target.value
+        })
+    }
+
+    render(){
+        // console.log(this)
+        const {value, caption} = this.state
+        return(
+            <form onSubmit={(e) => this.props.handleSubmit(this.state, e)}>
```

Comment:
This can be a function that takes in a handleSubmit from the Donations component